lib/honeybadger/utils.ex

```diff
@@ -17,4 +17,77 @@ defmodule Honeybadger.Utils do
     |> Module.split()
     |> Enum.join(".")
   end
+
+  @doc """
+  Transform value into a consistently cased string representation
+  """
+  def canonicalize(val) do
+    val
+    |> to_string()
+    |> String.downcase()
+  end
+
+  @doc """
+  Configurable data sanitization. This currently:
+
+  - recursively truncates deep structures (to a depth of 20)
+  - constrains large string values (to 64k)
+  - filters out any map keys that might contain sensitive information.
+  """
+  @default_max_string_size 65526
```

Comment:
This may be more obvious through multiplication:

```suggestion
  @default_max_string_size 64 * 1024
```